### PR TITLE
Soni Phonetic layout

### PR DESCRIPTION
**Pull request by arushidogra**

### CODE REVIEW

File: java/res/values/keyboard_names.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <string name="keyboard_name_bn_avro">বাংলা - Avro</string>
     <string name="keyboard_name_bn_inscript">বাংলা - Inscript</string>
     <string name="keyboard_name_bn_probhat">বাংলা - Probhat</string>
-
+    <string name="keyboard_name_soni_phonetic">Soni Phonetic</string>
```
Comment (jishnu7) [inline java/res/values/keyboard_names.xml]: this should be 
हिन्दी - Soni

File: java/res/values/keyboard_names.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <string name="keyboard_name_bn_avro">বাংলা - Avro</string>
     <string name="keyboard_name_bn_inscript">বাংলা - Inscript</string>
     <string name="keyboard_name_bn_probhat">বাংলা - Probhat</string>
-
+    <string name="keyboard_name_soni_phonetic">हिन्दी - Soni</string>
```
Comment (jishnu7) [inline java/res/values/keyboard_names.xml]: use keyboard_name_hi_soni

File: java/res/xml/method.xml [hunk 3]
```diff
@@ -285,7 +285,14 @@
             android:imeSubtypeLocale="si"
             android:imeSubtypeMode="keyboard"
             android:imeSubtypeExtraValue="KeyboardLayoutSet=qwerty,TransliterationMethod=si-singlish" />
-
+    
+    <!-- Soni Phonetic -->
+    <subtype android:icon="@drawable/ic_ime_switcher_dark"
+            android:label="@string/keyboard_name_soni_phonetic"
+            android:imeSubtypeLocale="soni_IN"
```
Comment (jishnu7) [inline java/res/xml/method.xml]: soni is not a locale. hindi is

**Comment by jishnu7** [comment]
changes for Santali is here again.

**Comment by jishnu7** [comment]
you need to update ime submodule which has soni layout

**Comment by arushidogra** [comment]
I did the changes.

**Pull request merged**